packages/panels/resources/views/components/resources/tabs.blade.php

```diff
@@ -0,0 +1,31 @@
+@if (count($tabs = $this->getCachedTabs()))
+    @php
+        $activeTab = strval($activeTab);
```

Comment:
ErrorException: `Undefined variable $activeTab`

Incomplete workaround from HasTabs trait: `$activeTab = $this->getDefaultActiveTab();`